src/vs/workbench/browser/parts/editor/multiEditorTabsControl.ts

```diff
@@ -132,6 +132,29 @@ export class MultiEditorTabsControl extends EditorTabsControl {
 	private lastMouseWheelEventTime = 0;
 	private isMouseOverTabs = false;
 
+	private _dndDropTarget: { leftElement: HTMLElement | undefined; rightElement: HTMLElement | undefined } | undefined;
```

Comment:
I would move this block closer to `updateDropFeedback` where we use it to better understand the use. I am not sure about the use of a setter here, I would probably call it `updateDropTarget` or similar and keep the current target in a variable if we have to.

Reply:
The reason I made it a setter is to make sure it would never change the value without removing the css classes. But I'm fine with creating a method